elastic/security/track.json

```diff
@@ -92,14 +92,6 @@
 {% endif %}
 {% endfor %}
   ],
-  {% if index_mode == "logsdb" %}
-  "component-templates": [
-    {
-      "name": "track-shared-logsdb-mode",
-      "template": "./templates/component/track-shared-logsdb-mode.json"
-    }
-  ],
-  {% else %}
```

Comment:
We need to keep this logic. Today we use `logsdb` index mode in internal benchmarks with `elastic/security` with the assumption that all the configuration is coming from Kibana, and we only overwrite the bare minimum to configure the index mode. cc @salvatore-campagna

Reply:
Restored this logic.